test/functionalTest/testHarness.sh

```diff
@@ -29,6 +29,20 @@ MAX_TRIES=${CB_MAX_TRIES:-3}
 echo $testStartTime > /tmp/brokerStartCounter
 
 
+# -----------------------------------------------------------------------------
+#
+# DISABLED - funct tests that are disabled, for some reason
+#
+DISABLED=('test/functionalTest/cases/1156_qfilters_and_compounds/qfilters_and_compounds_equals_null.test' \
```

Comment:
Alternative: to move the list to an external file (e.g. "DISABLED") in the same directory of testHarness.sh, so the program is not hardwiring any information.

That alternative seems to be smarter... but it adds another file to be aware of. Not sure what I prefer...

At the end, I think I prefer to keep as it is in the PR right now. What do you think?

Reply:
It is easy enough to modify the vector, as it is clearly marked in the beginning of the script.
Having it in another file ... more elegant but slightly more complicated.
I don't have a strong opinion on this, so, I prefer to keep it as is :-)

Reply:
NTC